rosdep/python.yaml

```
@@ -8962,6 +8962,15 @@ python3-twisted:
     '*': ['python%{python3_pkgversion}-twisted']
     '7': null
   ubuntu: [python3-twisted]
+python3-typeguard:
+  arch: [python-typeguard]
+  debian: [python3-typeguard]
+  fedora: [python3-typeguard]
```

The `Files changed` tab of this PR contains this suggestion:

https://packages.fedoraproject.org/pkgs/python-typeguard/python3-typeguard/

```suggestion
  fedora: [python3-typeguard]
  rhel:
    '*': [python3-typeguard]
    '7': null
```

---

Cool, thank you for that. I didn't understand what it was suggesting. I thought adding the package for fedora would also cover rhel if it has the same name.